x-pack/legacy/plugins/security/server/lib/authorization/check_saved_objects_privileges.test.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SpacesPlugin } from '../../../../spaces';
+import { OptionalPlugin } from '../../../../../server/lib/optional_plugin';
+import { checkSavedObjectsPrivilegesWithRequestFactory } from './check_saved_objects_privileges';
+
```

Comment:
These are modeled after the [`checkPrivilegesDynamicallyWithRequest` test suite](https://github.com/elastic/kibana/blob/94c3cb671947df421a28602af484f5aa6c291721/x-pack/legacy/plugins/security/server/lib/authorization/check_privileges_dynamically.test.ts)

Reply:
You know how much I like not having to think <3